### PR TITLE
Address accessible link and duplicate H1 issues.

### PR DESCRIPTION
**Pull request by cjoulain**
Here are some WCAG flags that I noticed. Details are courtesy of the [ARC](https://www.tpgi.com/arc-platform/arc-toolkit/), [WAVE](https://wave.webaim.org/extension/) [Axe](https://www.deque.com/axe/devtools/chrome-browser-extension/), and [Lighthouse](https://developer.chrome.com/docs/lighthouse/overview/) plug-ins on the Chrome and Firefox browsers.
## Short summary
- Use aria-label for SVG so that Homepage is narrated for screen reader users
- The content variable seems to include an H1 tag. Remove other H1, which also appears on the homepage
- Use underline CSS so that link is distinguishable without relying on color alone
- Screen readers are reading out both "navigation" (set by role) and "main navigation" (set by aria-label). We should remove one so it's not duplicative. 

## Long summary

[3.1.2 - Name, Role, Purpose](https://www.w3.org/TR/WCAG22/#name-role-value)

ARC shows aria-label name-role-purpose failure

The SVG icon visually displays a home icon. However, it does not communicate its purpose as a homepage to screen reader users. Images should generally have alt text or an aria-label to describe its meaning. For screen readers, it narrates as "www.zinc.coop". To make context clearer, since it's a homepage, we can use "Zinc homepage" or "homepage" as the text content. 

The following is from the NVDA screen reader. In addition to narrating web content in speech mode, NVDA also includes a document review, which is shown here. Notice that the log mentions the duplicate navigation/main navigation and that the home icon does not match the convention of the other categories (Product and Services, About Us, Join Us, Updates).

NVDA shows homepage has no description

[2.4.4 Link Purpose (In Context)](https://www.w3.org/TR/WCAG22/#link-purpose-in-context)

Technically `mailto:`creates a link so we'd like to include something text-based like an underline so color isn't the sole determining factor for a link. This will help low-vision and color-blind users.

There were two H1s on the default page. I tried tinkering with an "if title" statement but it didn't seem to work. The only action that removed the duplicate H1 was removing the H1 tag within main. I think this means the other H1 must be wrapped in the content variable and it seems like the content variable controls rendering of most of the body. Can someone confirm this is the correct action to take?

Duplicate H1s

## Before

WAVE results:
2 WAVE errors
Axe results:
2 Axe errors
Lighthouse results:
89% Lighthouse score

## After
WAVE results:
0 WAVE errors

Axe results:
0 Axe errors

Lighthouse results:
100% Lighthouse score

### CODE REVIEW

File: src/_includes/layouts/default.njk [hunk 1]
```diff
@@ -10,7 +10,6 @@
   <body>
     {% include "partials/header.njk" %}
     <main>
-      <h1>{{ title }}</h1>
```
Comment (anaulin) [inline src/_includes/layouts/default.njk]: Well spotted about the duplicate `h1` tags!

I think the solution is not to remove this line from the shared layout, though. This is where the "title" defined in each page goes, and if we remove it many pages will not have an `h1` tag at all. For example, https://www.zinc.coop/product-and-services/, defined in https://github.com/zinc-collective/www.zinc.coop/blob/development/src/product-and-services/index.md?plain=1, would no longer show the "Products and Services" title.

I think a better solution is to fix the buggy home page implementation: it is missing a "title" in its metadata, which causes this to be an empty `h1` tag, and then adds an extra top-level heading to compensate.

In short:
* restore this line in the shared template
* in https://github.com/zinc-collective/www.zinc.coop/blob/development/src/index.md?plain=1
  * add `title: The Zinc Collective` in the metadata section for the page
  * remove extra header from line 4
  
Optionally, it might be a good idea to double-check that all of our pages have a `title` defined.

Comment (cjoulain) [reply src/_includes/layouts/default.njk]: Thank you for the clear explanation, @anaulin! Added title back to settings. Created new endpoint that checks if there's a title. This is what it looks like locally:

List of missing titles

Comment (anaulin) [reply src/_includes/layouts/default.njk]: Neat!

**Review by anaulin — APPROVED** [review]
Thank you for these improvements!

File: src/_includes/partials/header.njk [hunk 2]
```diff
@@ -1,8 +1,8 @@
-<header class="header-container" role="navigation" aria-label="main navigation">
+<header class="header-container" aria-label="Main navigation">
 
   {# ----- Home Menu Item ----- #}
   <a class="header-item" href="/">
-    <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
+    <svg aria-label="Homepage" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
```
Comment (zspencer) [inline src/_includes/partials/header.njk:5]: 😻😻😻

File: src/a11y.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+---
+<p>There is a title missing from the following sections:</p>
+
+<ul>
+  {% for page in collections.all %}
+    {% if not page.data.title %}
+      <li>Missing title: {{ page.url }}</li>
+    {% endif %}
+  {% endfor %}
+</ul>
```
Comment (zspencer) [inline src/a11y.md:14]: This is a great idea! Thank you!

**Review by zspencer — APPROVED** [review]
🎉🎉🎉
